lib/task-data/schemas/install-ubuntu.json

```diff
@@ -68,6 +68,12 @@
         },
         "progressMilestones": {
             "$ref": "types-installos.json#/definitions/ProgressMilestones"
+        },
+        "timezone": {
+            "$ref": "types-installos.json#/definitions/Timezone"
+        },
+        "ntp": {
+            "$ref": "types-installos.json#/definitions/Ntp"
```

Comment:
install-debian.json is exactly the same with install-ubuntu.json, you should remove install-ubuntu.json and reference only install-debian.json in install debian/ubuntu tasks.